.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: rust
+rust:
+- nightly-2016-04-11
```

Comment:
@killercup Can you add plain `nightly` here as well so we can be notified when new nightlies breaks Miri?